cli/cfncluster/cfnconfig.py

```diff
@@ -268,47 +270,133 @@ def __init__(self, args):
             pass
 
         # Determine if EBS settings are defined and set section
+
+        # #self.parameters.append(("EBSSubstackURL",
+        #                         "https://s3.us-east-2.amazonaws.com/shuningc-cfncluster/extra_template/tropo_vol_substack.cfn.json"))
+
+        # CHANGED: adapt to list of ebs_settings
         try:
+            #see if there is a "ebs_settings" section, if there is no such section at all, give configparser.NoOptionError and pass
             self.__ebs_settings = __config.get(self.__cluster_section, 'ebs_settings')
+
+            #there is a ebs_setting section with some setting self.__ebs_settings, check if there is some self.__ebs_settings
```

Comment:
These comments were made when I just started so that I could understand the program, it should not be a part of the modification. I will take these comments out